app/src/main/res/values/strings.xml

```diff
@@ -90,6 +90,7 @@
     <string name="menu_page_new_tab">New tab</string>
     <string name="menu_page_reading_lists">Reading lists</string>
     <string name="menu_page_recently_viewed">Recently viewed</string>
+    <string name="menu_page_archive">Archive</string>
```

Comment:
Why singular? There may be several archive pages, and for this reason it’s usually in plural (cf. [Template:Archives on enwiki](https://en.wikipedia.org/wiki/Template:Archives)).